subsys/bluetooth/audio/Kconfig.mcs

```
@@ -51,6 +37,43 @@ config BT_MCC
 
 if BT_MCC
 
+config BT_MCC_MEDIA_PLAYER_NAME_MAX
+	int "Max length of media player name"
+	default 20
+	range 1 255
+	help
+	  Sets the maximum number of bytes (including the null termination) of
+	  the name of the media player.  If the name is longer, the media
```

```suggestion
	  the name of the media player. If the name is longer, the media
```
Here and also in the following configs

---

Fixed